docs/modify-integration.md

```diff
@@ -0,0 +1,24 @@
+## Modifying built-in integrations
+Sematext App Agent has built-in support for various application [integrations](https://github.com/sematext/sematext-agent-integrations). 
+You can modify the built-in integration YAMLs to suit your needs or fix any issues present in these YAMLs.
+These changes can be modifying existing YAML files ( for example, adding new metric or modifying the description of existing metric) 
+or adding a new YAML file. Assuming you have installed Sematext App Agent (open source or enterprise version) and
+set up the agent to monitor the application, following are the steps:
```

Comment:
the application => your application
, following are the steps => , the following are the steps to/for ________

Reply:
fixed. "following are the steps to modify metrics for a built-in integration:"